src/components/BottomSheet/BottomSheet.js

```diff
@@ -13,34 +14,31 @@ const BottomSheet = React.forwardRef((props, ref) => {
       onOpen={() => (props.onOpen ? props.onOpen() : {})}
       closeOnDragDown={true}
       closeOnPressMask={props.closeOnPressMask}
-      height={props.height ? props.height : 260}
+      height={props.height ? props.height : 355}
       customStyles={{
         container: styles.containerSheet(pullBottom),
-        draggableIcon: styles.draggableIcon,
+        draggableIcon: styles.draggableIcon
       }}>
-      <View style={{...styles.container, ...props.viewstyle}}>
-        {props.children}
-      </View>
+      <View style={{...styles.container, ...props.viewstyle}}>{props.children}</View>
     </RBSheet>
   );
 });
 
-BottomSheet.displayName = 'BottomSheet'
+BottomSheet.displayName = 'BottomSheet';
 
 export default BottomSheet;
 
 const styles = StyleSheet.create({
   container: {
-    paddingHorizontal: 20,
-    paddingTop: 20,
-    paddingBottom: 38,
+    paddingTop: dimen.normalizeDimen(10),
+    paddingHorizontal: dimen.normalizeDimen(20)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The paddingTop and paddingHorizontal styles have been changed. Ensure that these changes do not negatively impact the layout or appearance of the BottomSheet component across different screen sizes and resolutions.
```diff
-    paddingTop: 20,
-    paddingHorizontal: 20,
+    paddingTop: dimen.normalizeDimen(10),
+    paddingHorizontal: dimen.normalizeDimen(20)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->